setup.rst

```diff
@@ -72,6 +81,19 @@ to meet those requirements.
 
     If the installer doesn't work for you or doesn't output anything, make sure
     that the PHP `Phar extension`_ is installed and enabled on your computer.
+    In windows 10 with xampp, if you have un error like
```

Comment:
I think this is an error that occurs in all cases on Windows. So let's make a bit more generic:

```rst
.. note::

    If you're on Windows, you might get this error:

        cURL error 60: SSL certificate problem: unable to get local issuer certificate.

    In this case, download this link: http://curl.haxx.se/ca/cacert.pem and move the
    file on your computer. Then edit your `php.ini` file and insert:

    .. code-block:: ini

        curl.cainfo = "C:\path\to\cacert.pem"
```